components/tag/index.css

```diff
@@ -18,6 +18,7 @@ governing permissions and limitations under the License.
     filter: color;
     replace: --spectrum-tag$1;
   }
+  --spectrum-tag-texticon-text-font-size: var(--spectrum-global-dimension-font-size-75);
```

Comment:
Seems like this should be driven by tokens, is this missing from DSS?

Reply:
it is incorrect in DSS. I'll circle back and fix it afterwards; I'm mostly expediting this because it's blocking releases of other things like tabs.